src/Symfony/Component/Process/Process.php

```diff
@@ -1099,15 +1102,7 @@ public function getEnv()
      */
     public function setEnv(array $env)
     {
-        // Process can not handle env values that are arrays
-        $env = array_filter($env, function ($value) {
-            return !is_array($value);
-        });
-
-        $this->env = array();
-        foreach ($env as $key => $value) {
-            $this->env[$key] = (string) $value;
-        }
```

Comment:
In fact, I did it in this PR: keep only the array_filter part.
That will conflict with master so we'll have a reminder.